color/make_x11_colors.py

```diff
@@ -0,0 +1,269 @@
+    ("lightblue", (173, 216, 230, 255)),
+    ("lightcoral", (240, 128, 128, 255)),
+    ("lightcyan", (224, 255, 255, 255)),
+    ("lightgoldenrodyellow", (250, 250, 210, 255)),
+    ("lightgray", (211, 211, 211, 255)),
+    ("lightgreen", (144, 238, 144, 255)),
+    ("lightpink", (255, 182, 193, 255)),
+    ("lightsalmon", (255, 160, 122, 255)),
+    ("lightseagreen", (32, 178, 170, 255)),
+    ("lightskyblue", (135, 206, 250, 255)),
+    ("lightslategray", (119, 136, 153, 255)),
+    ("lightsteelblue", (176, 196, 222, 255)),
+    ("lightyellow", (255, 255, 224, 255)),
+    ("lime", (0, 255, 0, 255)),
+    ("limegreen", (50, 205, 50, 255)),
+    ("linen", (250, 240, 230, 255)),
+    ("magenta", (255, 0, 255, 255)),
+    ("maroon", (128, 0, 0, 255)),
+    ("mediumaquamarine", (102, 205, 170, 255)),
+    ("mediumblue", (0, 0, 205, 255)),
+    ("mediumorchid", (186, 85, 211, 255)),
+    ("mediumpurple", (147, 112, 219, 255)),
+    ("mediumseagreen", (60, 179, 113, 255)),
+    ("mediumslateblue", (123, 104, 238, 255)),
+    ("mediumspringgreen", (0, 250, 154, 255)),
+    ("mediumturquoise", (72, 209, 204, 255)),
+    ("mediumvioletred", (199, 21, 133, 255)),
+    ("midnightblue", (25, 25, 112, 255)),
+    ("mintcream", (245, 255, 250, 255)),
+    ("mistyrose", (255, 228, 225, 255)),
+    ("moccasin", (255, 228, 181, 255)),
+    ("navajowhite", (255, 222, 173, 255)),
+    ("navy", (0, 0, 128, 255)),
+    ("oldlace", (253, 245, 230, 255)),
+    ("olive", (128, 128, 0, 255)),
+    ("olivedrab", (107, 142, 35, 255)),
+    ("orange", (255, 165, 0, 255)),
+    ("orangered", (255, 69, 0, 255)),
+    ("orchid", (218, 112, 214, 255)),
+    ("palegoldenrod", (238, 232, 170, 255)),
+    ("palegreen", (152, 251, 152, 255)),
+    ("paleturquoise", (175, 238, 238, 255)),
+    ("palevioletred", (219, 112, 147, 255)),
+    ("papayawhip", (255, 239, 213, 255)),
+    ("peachpuff", (255, 218, 185, 255)),
+    ("peru", (205, 133, 63, 255)),
+    ("pink", (255, 192, 203, 255)),
+    ("plum", (221, 160, 221, 255)),
+    ("powderblue", (176, 224, 230, 255)),
+    ("purple", (128, 0, 128, 255)),
+    ("rebeccapurple", (102, 51, 153, 255)),
+    ("red", (255, 0, 0, 255)),
+    ("rosybrown", (188, 143, 143, 255)),
+    ("royalblue", (65, 105, 225, 255)),
+    ("saddlebrown", (139, 69, 19, 255)),
+    ("salmon", (250, 128, 114, 255)),
+    ("sandybrown", (244, 164, 96, 255)),
+    ("seagreen", (46, 139, 87, 255)),
+    ("seashell", (255, 245, 238, 255)),
+    ("sienna", (160, 82, 45, 255)),
+    ("silver", (192, 192, 192, 255)),
+    ("skyblue", (135, 206, 235, 255)),
+    ("slateblue", (106, 90, 205, 255)),
+    ("slategray", (112, 128, 144, 255)),
+    ("snow", (255, 250, 250, 255)),
+    ("springgreen", (0, 255, 127, 255)),
+    ("steelblue", (70, 130, 180, 255)),
+    ("tan", (210, 180, 140, 255)),
+    ("teal", (0, 128, 128, 255)),
+    ("thistle", (216, 191, 216, 255)),
+    ("tomato", (255, 99, 71, 255)),
+    ("transparent", (0, 0, 0, 0)),
```

Comment:
`transparent` is a special-case in `parse::parse_color`. If that remains, this row can be dropped and then alpha does not need to be stored.

https://github.com/linebender/color/blob/e824f679228476722275a25d2107635743b5a33e/color/src/parse.rs#L376-L383

Reply:
I think this same list will be used to generate a bunch of const values like `color::named::TRANSPARENT` or whatever, and then we would want it here.

Reply:
Special casing `TRANSPARENT` in that case would also be fine IMO, so shouldn't be a blocker for this suggestion. Of note, [`bevy_color`](https://docs.rs/bevy_color/latest/bevy_color/palettes/index.html) doesn't provide the transparent colour.

Reply:
I've deliberately moved it out, partly as a performance optimization, partly because transparent is an expected palette color. I suppose it could save 142 bytes, as the 3-byte colors would be stored packed, but I'm not sure it's worth it.